### DISCUSSION TITLE
Version 0.5.3 (final)

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1518] [Announcements]
## 1. Features:
* None.  This is a patch release to resolve multiple issues with application lockup, buffer overflows and excessive memory usage.

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. What's Changed
* #1405 Memory/Thread leak when using Polyphase Channelizer by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1477

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.5.1...v0.5.2

## What's Changed
* #1485 Sets Blackman-Harris 7 As Default FFT Window by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1486
* #1487 Thread Locking Impacting Processing Workload by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1488
* #1491 Polyphase Channellizer - Add Diagnostic Logging to Dispatcher Thread by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1492
* #1498 JMBE Audio Class Loading - Memory Leak by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1499
* #1506 Resolve App Lockup, Tuner Freezes, Buffer Overflows & Excessive Memory Usage by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1513

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.5.2...v0.5.3

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/v0.5.3'>Version 0.5.3 (final)</a>.</em>

**Comment by Clark985** [comment]
This fixed the app lockup issues I was having on multiple receive PCs of mine, thanks!
